core/src/testFixtures/java/io/grpc/internal/AbstractTransportTest.java

```diff
@@ -139,9 +139,6 @@ protected abstract InternalServer newServer(
   /**
    * Returns true (which is default) if the transport reports message sizes to StreamTracers.
```

Comment:
Delete the javadoc for the deleted method as well.

Reply:
Oh okay, this was left unseen. Done.